README.md

```diff
@@ -1,2 +1,40 @@
-# onlineshop;
-welcome
+# online-shop
+![Language grade: Java](images/onlinestore.jpg)
```

Comment:
check if this image is showing correctly

Reply:
correctly